rec_to_nwb/processing/builder/originators/position_originator.py

```diff
@@ -53,15 +53,24 @@ def make(self, nwb_content: NWBFile):
                 position_df = self.get_position_with_corrected_timestamps(
                     position_tracking_path[0]
                 )
-                position.create_spatial_series(
-                    name=f"series_{dataset_ind}",
-                    description=", ".join(position_df.columns.tolist()),
-                    data=np.asarray(position_df),
-                    conversion=conversion,
-                    reference_frame="Upper left corner of video frame",
-                    timestamps=np.asarray(position_df.index),
-                )
-                first_timestamps.append(position_df.index[0])
+                #Multi-position split. 
+                #TODO: generalize key names?
+                key_lists = [['xloc','yloc','zloc'],
+                             ['xloc2','yloc2','zloc2'],
+                             ['xloc3','yloc3','zloc3']]
+                led_number = 0
+                for led_number, valid_keys in enumerate(key_lists):
+                    key_set = [key for key in position_df.columns.tolist() if key in valid_keys]
+                    if len(key_set) > 0:
+                        position.create_spatial_series(
+                            name=f"led_{led_number}_series_{dataset_ind}",
+                            description=", ".join(key_set),
+                            data=np.asarray(position_df[key_set]),
+                            conversion=conversion,
+                            reference_frame="Upper left corner of video frame",
+                            timestamps=np.asarray(position_df.index),
+                        )
+                        first_timestamps.append(position_df.index[0])
```

Comment:
I think we can take the minimum of the two spatial series since this is just used to check if the position data is in correct order.

Reply:
So only append it once per epoch?

Reply:
Right. Just take the earliest time (which for all intents and purposes should be the same for both spatial series anyways).